portals/application/configs/klear/ExternalCallFiltersRetailList.yaml

```diff
@@ -0,0 +1,99 @@
+#include conf.d/mapperList.yaml
+#include conf.d/actions.yaml
+
+# ************************ ExternalCallFiltersList.yaml ************************
+production:
+  main:
+    module: klearMatrix
+    defaultScreen: externalCallFiltersList_screen
+  screens: &externalCallFilters_screensLink
```

Comment:
I would keep unique names as long as it's not a trouble: externalCallFiltersRetailList_screensLink